src/Resources/config/services.yml

```diff
@@ -27,6 +27,18 @@ services:
         tags:
             - { name: kernel.cache_warmer }
 
+    contao.controller.abstract_csv_import:
```

Comment:
Ah different comment here :D Yeah it's pretty unlikely that I, myself, would use this service definition when writing my own service. Not needed imho, indeed. Makes ready the services.yml just more complicated for no real reason.